docker_run_pg_tests.sh

```diff
@@ -0,0 +1,10 @@
+export PGDATA=/var/lib/postgresql/data
```

Comment:
could do with a shebang line and a comment saying what it does?

and a `set -e` maybe?

Reply:
can it go in a subdir?

Reply:
done